src/Cli/dotnet/commands/dotnet-test/TestingPlatformCommand.cs

```diff
@@ -174,6 +227,16 @@ private Task<IResponse> OnRequest(IRequest request)
             return Task.FromResult((IResponse)VoidResponse.CachedInstance);
         }
 
+        private static HandshakeInfo CreateHandshakeInfo() =>
+            new(new Dictionary<string, string>
+            {
+                { HandshakeInfoPropertyNames.PID, Process.GetCurrentProcess().Id.ToString() },
```

Comment:
Similarly to MSTest repo, call to `.Id` can throw https://learn.microsoft.com/dotnet/api/system.diagnostics.process.id?view=net-8.0#exceptions so you most likely want to do some `try/catch (IOE)`

Reply:
@MarcoRossignoli said there should be no problem here so feel free to ignore this case